### DISCUSSION TITLE
How to render JSON columns

### DISCUSSION THREAD

**Discussion by faridsa** [rappasoft/laravel-livewire-tables#856] [Q&A] [answered]
Is there a way to render mysql JSON columns as json_decode instead of showing raw content?
I'm using v2.7. Thank you in advance.

**Comment by dgillier** [comment, marked as answer]
You can use the format option like in :

```
           Column::make('Name", "data")
                ->format(fn($value, $row, $column) => json_decode($value)),
```

**Reply by faridsa** [reply]
Thank you so much. Even though using the function `json_decode()` generates an error
`htmlspecialchars(): Argument #1 ($string) must be of type string`. Your answer gave me the idea to use `str_replace()` and with it I achieve the desired result.

**Reply by dgillier** [reply]
You can also format value as html like in:

```
 Column::make('Name", "data")
                ->format(fn($value, $row, $column) => json_decode($value))
                ->html(),
```
